### PR TITLE
Update available JDK versions

### PR DESCRIPTION
**Pull request by wychen**
PTAL

### CODE REVIEW

File: user/reference/bionic.md [hunk 1]
```diff
@@ -123,8 +123,7 @@ If you're getting errors about PyPy `pypy is not installed; attempting download`
 
 ## JVM (Clojure, Groovy, Java, Scala) support
 
-* Pre-installed JVMs: `openjdk8`, `openjdk10`, and `openjdk11` on x86, default
```
Comment (micolous) [inline user/reference/bionic.md]: `bionic` [also supports](https://travis-ci.org/micolous/metrodroid/jobs/564284797#L176) `openjdk9`.

**Comment by Pezi777** [comment]
hey @wychen,
thank you so much for creating this pull request. The engineer who has worked on the Bionic project specified which versions come pre-installed (see https://github.com/travis-ci/docs-travis-ci-com/pull/2454) and the others will then be available on demand.
Cheers!

**Comment by wychen** [comment]
I was trying to get JDK 8 there, but looks like the lowest supported version is 9. Might be useful to include that info in the doc as well.

**Comment by schildbach** [comment]
I would really like to get OpenJDK 8 with the Bionic image. It shouldn't be a great deal since Ubuntu Bionic comes with openjdk-8-jdk. JDK 8 is still necessary for Android development.

**Comment by mike-ross-88** [comment]
Please add openjdk8 support, having to fallback to Ubuntu Xenial is less than ideal.

**Comment by lbergelson** [comment]
This is a surprising pain point for us.  It's annoying considering java 8 is still by far the most widely used and is available as a bionic package.   By one metric java 8 is 79% of java usage as of october 2019 which is the most recent data I can find https://www.baeldung.com/java-in-2019.

**Comment by Pezi777** [comment]
openjdk8 is now available @mike-ross-88

**Comment by lbergelson** [comment]
@Pezi777 That's great news, thank you!

**Pull request closed without merging**